s3/pkg/datastore/azure/azure.go

```diff
@@ -197,7 +204,7 @@ func (ad *AzureAdapter) Copy(ctx context.Context, stream io.Reader, target *pb.O
 func (ad *AzureAdapter) ChangeStorageClass(ctx context.Context, object *pb.Object, newClass *string) error {
 	objectId := object.ObjectId
 	blobURL := ad.containerURL.NewBlockBlobURL(objectId)
-	log.Infof("change storage class[Azure Blob], objectId:%s, blobURL is %v\n", objectId, blobURL)
+	log.Infof("change storage class[Azure Blob], objectId:%s, storageClass:%s blobURL is %v\n", objectId, *newClass, blobURL)
```

Comment:
newClass variable is a pointer of string which is kind of abuse of string in golang because "In Go, a string is in effect a read-only slice of bytes" which intern means its pointer already.

Reply:
done. #1201